CHANGELOG.md

```diff
@@ -10,9 +10,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 
 - Added support for xlsx exports in Unchained Lunatic parser
+
 ### Fixed
 
+- Fix SoftMax Pro case where some columns were incorrectly identified as not numeric for calculated data documents
```

Comment:
You can probably specify the values of `Masked` and `Range`  here too? what do you think?